apis/kms/v1alpha1/generator-config.yaml

```diff
@@ -0,0 +1,7 @@
+ignore:
+  resource_names:
+  - CustomKeyStore
+  - Grant
+  - Alias
+  field_paths:
+  - CreateAliasInput.TargetKeyId
```

Comment:
```suggestion
```

I couldn't find a place where `CreateAliasInput` would be used by ACK since `Alias` is already ignored.